oggm/tests/test_utils.py

```diff
@@ -323,6 +323,26 @@ def test_clips_performance(self):
         t2 = timeit.timeit('utils.clip_min(a, 15)', number=n, setup=s2)
         assert t2 < t1
 
+    def test_cook_rgidf(self):
+        from oggm import workflow
+        cfg.initialize()
+        cfg.PARAMS['use_intersects'] = False
+        cfg.PATHS['working_dir'] = utils.mkdir(utils.get_temp_dir(dirname='test_cook_rgidf'),
+                                               reset=True)
+        path = utils.get_demo_file('cgi2.shp')
+        cgidf = gpd.read_file(path)
+        rgidf = utils.cook_rgidf(cgidf, save_special_columns={'Glc_Long': 'CenLon',
+                                                              'Glc_Lati': 'CenLat'},
+                                 assign_col_values={'Area': cgidf.Glc_Area*1e-6})
+        gdirs = workflow.init_glacier_directories(rgidf)
```

Comment:
Yes so I think you need to stop the test here, and ensure that OGGM could still create the glacier directories properly (that's the main test anyway)